physics/biot_savart_spacepy.py

```diff
@@ -39,56 +39,26 @@
 
 files = util.filelist()
 
-n = files.size
+print files[0]
+n = len(files)
 n = 2
 for i in range(n):
     filename_split = files[i]
 
-    #time = data[i, 0:5]
-    mlat = data[i, 5]
-    mlon = data[i, 6]
-    #Event = data[i, :]
-
-    time = util.filename2time(filename_split)
+    time = util.filename2time(filename_split) # returns length 7 list
 
     # get list of events with same y,m,d,hour,min
-    events.events(time[0:6))
-
-    #filename_split = events.findfile(time)
+    events_array = events.events(time[0:5])
+    print(events_array.shape)
+    if events_array.size == 0:
+        print('hello there')
+        continue
 
-    filename = conf['run_path'] + filename_split
+    filename = conf['run_path'] + filename_split[0:34] # remove '.cdf'
```

Comment:
Won't work if filename changes. Use `replace` or https://kite.com/python/examples/4300/os-remove-the-file-extension-from-a-filename